open-media-match/src/OpenMediaMatch/__init__.py

```diff
@@ -5,38 +5,40 @@
 
 from .blueprints import hashing, matching, curation
 
+
 def create_app():
     """
     Create and configure the Flask app
     """
     app = flask.Flask(__name__)
-    app.config.from_envvar('OMM_CONFIG')
-    
-    @app.route('/')
+    app.config.from_envvar("OMM_CONFIG")
+
+    @app.route("/")
```

Comment:
@dougneal  - this is going to create collisions with your outstanding diffs, but there's no way around it, and we should be innoculated going forward.

Reply:
It's ok